simples3.go

```diff
@@ -41,6 +42,41 @@ type S3 struct {
 	URIFormat string
 }
 
+type ListObjectsV2Details struct {
+	Bucket            string
+	Delimiter         string
+	ContinuationToken string
+	EncodingType      string
+	FetchOwner        string
+	MaxKeys           string
+	Prefix            string
+	StartAfter        string
+}
+
+// ListObjectsV2Response is passed to ListObjectsV2 as a parameter.
+type ListObjectsV2Response struct {
+	Name        string `xml:"Name"`
+	Prefix      string `xml:"Prefix"`
+	KeyCount    string `xml:"KeyCount"`
+	MaxKeys     string `xml:"MaxKeys"`
+	Delimiter   string `xml:"Delimiter"`
+	IsTruncated string `xml:"IsTruncated"`
+	Contents    []struct {
```

Comment:
Should this be extracted into a named struct?

Reply:
Unsure. What's your preference?